pkg/cmd/serviceaccount/phases/create/federatedidentitycredential.go

```diff
@@ -66,6 +67,7 @@ func (p *federatedIdentityPhase) run(ctx context.Context, data workflow.RunData)
 
 	serviceAccountNamespace, serviceAccountName := createData.ServiceAccountNamespace(), createData.ServiceAccountName()
 	subject := util.GetFederatedCredentialSubject(serviceAccountNamespace, serviceAccountName)
+	name := strings.Join([]string{createData.ServiceAccountNamespace(), createData.ServiceAccountName(), util.GetIssuerHash(createData.ServiceAccountIssuerURL())}, "-")
```

Comment:
Service account name can be at most 253 chars and namespace can be at most 63 chars. All this compounded can be long in the max scenario.

Not a blocker for this PR but I think we should follow up on the char limitation for federated identity credential name.